spec/acceptance/settings_spec.rb

```diff
@@ -87,6 +87,95 @@ class { 'pulpcore':
   end
 end
 
+describe 'AUTHENTICATION_BACKENDS setting' do
+  context 'default AUTHENTICATION_BACKENDS' do
+    it_behaves_like 'an idempotent resource' do
+      let(:manifest) do
+        <<-PUPPET
+        include pulpcore
+        PUPPET
+      end
+    end
+
+    describe file('/etc/pulp/settings.py') do
+      it { is_expected.to be_file }
+      its(:content) { is_expected.to include('AUTHENTICATION_BACKENDS = ["pulpcore.app.authentication.PulpNoCreateRemoteUserBackend"]') }
+    end
+  end
+
+  context 'AUTHENTICATION_BACKENDS set' do
+    it_behaves_like 'an idempotent resource' do
+      let(:manifest) do
+        <<-PUPPET
+        class { 'pulpcore':
+          authentication_backends => [
+            'django.contrib.auth.backends.ModelBackend',
+            'pulpcore.app.authentication.PulpNoCreateRemoteUserBackend',
+          ],
+        }
+        PUPPET
+      end
+    end
+
+    describe file('/etc/pulp/settings.py') do
+      it { is_expected.to be_file }
+      its(:content) { is_expected.to include('AUTHENTICATION_BACKENDS = ["django.contrib.auth.backends.ModelBackend", "pulpcore.app.authentication.PulpNoCreateRemoteUserBackend"]$') }
+    end
+  end
+end
+
+describe 'REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES setting' do
+  context 'default REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES' do
+    it_behaves_like 'an idempotent resource' do
+      let(:manifest) do
+        <<-PUPPET
+        include pulpcore
+        PUPPET
+      end
+    end
+
+    describe file('/etc/pulp/settings.py') do
+      it { is_expected.to be_file }
+      its(:content) do
+        is_expected.to include <<~EXPECTED
+          REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES = (
+              'rest_framework.authentication.SessionAuthentication',
+              'pulpcore.app.authentication.PulpRemoteUserAuthentication',
+          )
+        EXPECTED
+      end
+    end
+  end
+
+  context 'REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES set' do
+    it_behaves_like 'an idempotent resource' do
+      let(:manifest) do
+        <<-PUPPET
+        class { 'pulpcore':
+          rest_framework_default_authentication_classes => [
+            'rest_framework.authentication.BasicAuthentication',
+            'rest_framework.authentication.SessionAuthentication',
+            'pulpcore.app.authentication.PulpRemoteUserAuthentication'
```

Comment:
Wouldn't we expect a trailing comma here?
```suggestion
            'pulpcore.app.authentication.PulpRemoteUserAuthentication',
```